dipy/workflows/vis_registeration.py

```diff
@@ -0,0 +1,278 @@
+            Filename to be used for saving the mosaic (default 'mosaic.png').
+        """
+
+        overlay, value_range = self.process_image_data(static_img,
+                                                       moved_img)
+
+        renderer = window.Renderer()
+        renderer.background((0.5, 0.5, 0.5))
+
+        slice_actor = actor.slicer(overlay, affine, value_range)
+        renderer.projection('parallel')
+
+        cnt = 0
+        X, Y, num_slices = slice_actor.shape[:3]
+
+        rows, cols = self.get_row_cols(num_slices)
+        border = 5
+
+        for j in range(rows):
+            for i in range(cols):
+                slice_mosaic = slice_actor.copy()
+                slice_mosaic.display(None, None, cnt)
+                slice_mosaic.SetPosition((X + border) * i,
+                                         0.5 * cols * (Y + border) -
+                                         (Y + border) * j, 0)
+                slice_mosaic.SetInterpolate(False)
+                renderer.add(slice_mosaic)
+                renderer.reset_camera()
+                renderer.zoom(1.6)
+                cnt += 1
+                if cnt > num_slices:
+                    break
+            if cnt > num_slices:
+                break
+
+        renderer.reset_camera()
+        renderer.zoom(1.6)
+
+        window.record(renderer, out_path=fname,
+                      size=(900, 600), reset_camera=False)
+
+    def adjust_color_range(self, img):
+
+        """
+        Function to adjust the range of colors in numpy array
+        to create the GIF (GIF standard only supports 256 colours).
+
+        return
+        The numpy array with scaled down range of color values.
+        """
+
+        # Interpolating to range 0-15 for reducing the colours.
+        img[..., 0] = np.interp(img[..., 0], (img[..., 0].min(),
+                                              img[..., 0].max()), (0, 14))
+        img[..., 1] = np.interp(img[..., 1], (img[..., 1].min(),
+                                              img[..., 1].max()), (0, 14))
+        img = np.round(img, 0).astype('uint8')
+
+        # Interpolating back to the range 0-255 for getting GIF range.
+        img[..., 0] = np.interp(img[..., 0], (img[..., 0].min(),
+                                              img[..., 0].max()), (0, 255))
+        img[..., 1] = np.interp(img[..., 1], (img[..., 1].min(),
+                                              img[..., 1].max()), (0, 255))
+
+        img = np.round(img, 0).astype('uint8')
+
+        return img
+
+    def animate_overlap_with_renderer(self, static_img, moved_img,
+                                      sli_type, fname, affine):
+
+        """
+        Function for creating the animated GIF from the slices of the
+        registered image.
+        It uses the renderer object to control the dimensions of the
+        created GIF and for correcting the orientation.
+        Patameters
+        ----------
+        sli_type : str (optional)
```

Comment:
You're only saving 2 characters. It's not worth it. Can you please name it `slice_type`?

Reply:
Yes, you are right, though, I was already using the slice_type in the function (for storing the integer indices for the slice type and so sli_type was just a string name for the convenience of the user). Now, I am just overriding the variable. I did not know that I can also override it. I tried it now and it is working. Thank you.